pages/community/tsc.js

```diff
@@ -0,0 +1,271 @@
+            project. No matter if you are an individual contributor or backed by
+            a company, you have equal rights. Read{" "}
+            <a
+              href="https://www.asyncapi.com/blog/governance-motivation"
+              className="text-blue-500 hover:text-blue-400"
+            >
+              this
+            </a>{" "}
+            article in case you want to learn more.
+          </p>
+        </div>
+      </div>
+      <div className="mt-5">
+        <h3 className="font-semibold mb-5 text-primary-800 lg:text-2xl">
+          The current TSC members
+        </h3>
+
+        <ul
+          role="list"
+          className="space-y-4 sm:grid sm:grid-cols-2 sm:gap-6 sm:space-y-0 lg:grid-cols-3 lg:gap-8"
+        >
+          {tscMembers.map((user) => displayUserInfo(user))}
+          <QuestionCard />
+        </ul>
+      </div>
+    </GenericLayout>
+  );
+}
+
+function displayUserInfo(user) {
+  return (
+    <li
+      key={user.name}
+      className="p-4 text-center border rounded-md border-gray-200 shadow-md"
+    >
+      <div className="flex flex-row">
+        <img
+          src={user.avatarUrl}
+          className="mx-auto rounded-full h-20 w-20 xl:w-28 xl:h-28"
+        />
+        <div className="flex-1">
+          <div className="font-bold text-lg my-3">{user.name}</div>
+          {showWorkStatus(user)}
+          <ul role="list" className="flex justify-center space-x-5 my-5">
+            <li>
+              <a
+                href={user.github}
+                className="text-gray-600 hover:text-gray-500"
+              >
+                <span className="sr-only">Github</span>
+                <GithubSVG />
+              </a>
+            </li>
+            {user.twitter ? (
+              <li>
+                <a
+                  href={user.twitter}
+                  className="text-gray-600 hover:text-gray-500"
+                >
+                  <span className="sr-only">Twitter</span>
+                  <TwitterSVG />
+                </a>
+              </li>
+            ) : null}
+            {user.linkedin ? (
+              <li>
+                <a
+                  href={user.linkedin}
+                  className="text-gray-600 hover:text-gray-500"
+                >
+                  <span className="sr-only">LinkedIn</span>
+                  <LinkedInSVG />
+                </a>
+              </li>
+            ) : null}
+          </ul>
+        </div>
+      </div>
+      <div className="flex flex-wrap gap-1 items-center">
+        Maintainer of:
+        {user.repos.map((repo) => (
+          <a
+            key={repo.name}
+            className="inline-flex items-center px-3 py-0.5 rounded-full text-xs font-medium leading-5 bg-teal-100 text-teal-800 hover:bg-teal-300"
+            href={repo.url}
+          >
+            {repo.name}
+          </a>
+        ))}
+      </div>
+    </li>
+  );
+}
+
+function showWorkStatus(user) {
```

Comment:
Please change it to the React component. It's not a good pattern to write components as normal JS function.

```suggestion
function UserWorkStatus({ user }) {
```